package/system-role-kvm.spec

```diff
@@ -36,7 +36,7 @@ BuildRequires:  yast2-installation-control >= 4.0.0
 
 Url:            https://github.com/yast/system-role-kvm
 AutoReqProv:    off
-Version:        15.1.2
+Version:        15.1.3
```

Comment:
Hi, yast/products control/system roles does not follow symver but product based version. So if target of this change is SLE 15 SP3 then proper version is 15.3.0 ( so SLE15 SP3 0 as first version in given code stream ).

Reply:
But beside this, rest of change looks good.